test/built-ins/RegExp/prototype/flags/this-val-regexp-prototype.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-get-regexp.prototype.flags
+description: >
+  Return "" when the `this` value is the RegExp.prototype object
+info: |
+  1. Let R be the this value.
+  2. If Type(R) is not Object, throw a TypeError exception.
```

Comment:
I agree, but considering the delay, I believe it's best to fix this with follow ups.